packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -102,29 +118,42 @@ mkdir -p %{buildroot}/%{_prefix}/lib/gofer/plugins
 cp etc/gofer/plugins/katelloplugin.conf %{buildroot}/%{_sysconfdir}/gofer/plugins
 cp src/katello/agent/katelloplugin.py %{buildroot}/%{_prefix}/lib/gofer/plugins
 
-mkdir -p %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/package_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/enabled_repos_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+%if %{dnf_install}
+%define katello_libdir %{python3_sitelib}/katello
+%define plugins_dir %{python3_sitelib}/dnf-plugins
+%define plugins_confdir %{_sysconfdir}/dnf/plugins
+%else
+%define katello_libdir %{python_sitelib}/katello
+%define plugins_dir %{_prefix}/lib/yum-plugins
+%define plugins_confdir %{_sysconfdir}/yum/pluginconf.d
+%endif
+
+mkdir -p %{buildroot}/%{katello_libdir}
+mkdir -p %{buildroot}/%{plugins_dir}
+mkdir -p %{buildroot}/%{plugins_confdir}
+
+cp src/katello/*.py %{buildroot}/%{katello_libdir}/
```

Comment:
I realize this was already present in the old setup, but `%{katello_libdir}` is already an absolute path so the `/` between the two macros is not needed. Doesn't hurt, but it's cleaner not to. This is true in multiple places and I won't comment on them all but basically grep for `%{buildroot}/`.